sdk/src/schema/activity.ts

```diff
@@ -9,7 +9,7 @@ export type DirectusActivity<Schema extends object> = MergeCoreCollection<
 		id: number;
 		action: string;
 		user: DirectusUser<Schema> | string | null;
-		timestamp: string;
+		timestamp: 'datetime'; // string;
```

Comment:
Do we need this comment here considering that it is not present in other 'datetime' fields?
```suggestion
		timestamp: 'datetime';
```